mesa-quilt

```diff
@@ -88,6 +88,11 @@ if [ -d "$d/nx-X11/extras" ]; then
 
 		fi
 
+		# remove the .applied-patches file, if empty again
+		if [ ! -s "../Mesa.patches/.applied-patches" ]; then
+			rm "../Mesa.patches/.applied-patches"
```

Comment:
Can fail if the file does not exist, use `rm -f` instead to ignore a nonexistent file.